### ISSUE TITLE
loudRejection/api is deprecated

### ISSUE THREAD

**Issue by robinedman** [britishcouncil/react-quiz#13]
Running the tests outputs:

```
loudRejection/api is deprecated. Use the currently-unhandled module instead.
```

Upgrading AVA to the newest version should sort it out (https://github.com/avajs/ava/issues/900).

**Comment by dakala** [comment]
@robinedman See PR https://github.com/britishcouncil/react-quiz/pull/14

**Comment by robinedman** [comment]
Solved by #14
